src/k8s/cmd/k8s/k8s_config.go

```diff
@@ -25,4 +31,6 @@ func newKubeConfigCmd() *cobra.Command {
 			return nil
 		},
 	}
+	configCmd.PersistentFlags().StringVar(&configCmdOpts.server, "server", "", "Specify a custom cluster server address for the kubeconfig")
```

Comment:
nit: should be lower-case according to https://github.com/canonical/k8s-snap/pull/188
"flag descriptions should start with lowercase and should not end with a '.'"